CMakeLists.txt

```diff
@@ -1,22 +1,28 @@
-cmake_minimum_required(VERSION 3.0.2)
+cmake_minimum_required(VERSION 3.15)
 project(bio_ik)
 
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 20)
+endif()
```

Comment:
We can achieve the same functionality in a more expressive way with `target_compile_features` and the `cxx_std_20` metafeature. it lets users still specify CMAKE_CXX_STANDARD if they really want to but otherwise lets CMake automatically do whatever is right for a given compiler.

https://cmake.org/cmake/help/latest/prop_gbl/CMAKE_CXX_KNOWN_FEATURES.html

Reply:
Addressed in [340d3b2](https://github.com/PickNikRobotics/bio_ik/pull/2/commits/340d3b22ebc1a3948c0b48f6fc6a12e27c14f795)